libpromises/mod_custom.c

```diff
@@ -411,17 +411,24 @@ static void PromiseModule_DestroyInternal(PromiseModule *module)
 
 static PromiseModule *PromiseModule_Start(char *interpreter, char *path)
 {
-    assert(interpreter != NULL);
     assert(path != NULL);
 
-    if (access(interpreter, X_OK) != 0)
+    if ( (interpreter != NULL) && (access(interpreter, X_OK) != 0) )
```

Comment:
Please remove the spaces inside parentheses. We don't use this pattern, AFAICT. Applies to the other `if`s below as well.